SU-HSDO/suhumsci
# feat(STN-70): Decanter lockups

# READY FOR REVIEW

## Summary

[JIRA CARD](https://sparkbox.atlassian.net/browse/STN-70)

Adds Decanter lockup html and styles, as well as theme settings for Lockup options, and allows that global lockup variable be available to the correct template.

For expected implementation, take a look at Decanter: https://decanter.stanford.edu/component/identity-lockup/

### Screenshots

See my comment on the JIRA card for screenshots of the Lockup Options A-T, and default (no specific class) implementations.

## Steps to Test
1. Go to Admin -> Appearance -> Settings -> Humsci_Colorful and look for "Theme specific settings" -> "Lockup Settings"
2. Select an option, and then add 5 lines for your lockup text.
3. Take a look at your main page and see that some different options look correct, referencing Decanter. Note: "m" is the only option that appears a bit different, but I checked the styles, and everything is working the right way.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
